client/extensions/wp-super-cache/wrap-settings-form.jsx

```diff
@@ -50,16 +93,22 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 				handleToggle: this.handleToggle,
 			};
 
-			return <SettingsForm { ...this.props } { ...utils } />;
+			return (
+				<div>
+					<QuerySettings siteId={ this.props.siteId } />
+					<SettingsForm { ...this.props } { ...utils } />
+				</div>
+			);
 		}
 	}
 
 	const connectComponent = connect(
-		() => {
-			// TODO: Replace with REST API once ready.
-			const settings = {
+		state => {
+			const siteId = getSelectedSiteId( state );
+			const settings = Object.assign( {}, getSettings( state, siteId ), {
 				// Easy
-				scrules: true,
+				// http_only setting is not persisted, but is needed for the "Test Cache" action.
+				http_only: true,
```

Comment:
No, I mean getSettings() should likely set the defaults before it enters into any view, that way different views don't accidentally get different defaults set by accident. :)